<?xml version="1.0" ?>
<!-- =================================================================================== -->
<!-- |    This document was autogenerated by xacro from car.xacro                      | -->
<!-- |    EDITING THIS FILE BY HAND IS NOT RECOMMENDED                                 | -->
<!-- =================================================================================== -->
<robot name="satellite">
  <link name="car">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="1 0.5 0.25"/>
      </geometry>
      <material name="gray">
        <color rgba="0.7 0.7 0.7 1"/>
      </material>
    </visual>
    <inertial>
      <mass value="100"/>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <inertia ixx="2.0833333333333335" ixy="0" ixz="0" iyy="4.166666666666667" iyz="0" izz="2.0833333333333335"/>
    </inertial>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <box size="1 0.5 0.25"/>
      </geometry>
    </collision>
  </link>
  <link name="front_wheelbase"/>
  <link name="front_wheel_left">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.2" radius="0.25"/>
      </geometry>
      <material name="dark_gray">
        <color rgba="0.6 0.6 0.6 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.2" radius="0.25"/>
      </geometry>
    </collision>
  </link>
  <link name="front_wheel_right">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.2" radius="0.25"/>
      </geometry>
      <material name="dark_gray">
        <color rgba="0.6 0.6 0.6 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.2" radius="0.25"/>
      </geometry>
    </collision>
  </link>
  <link name="rear_wheelbase"/>
  <link name="rear_wheel_left">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.2" radius="0.25"/>
      </geometry>
      <material name="dark_gray">
        <color rgba="0.6 0.6 0.6 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.2" radius="0.25"/>
      </geometry>
    </collision>
  </link>
  <link name="rear_wheel_right">
    <visual>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.2" radius="0.25"/>
      </geometry>
      <material name="dark_gray">
        <color rgba="0.6 0.6 0.6 1"/>
      </material>
    </visual>
    <collision>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <geometry>
        <cylinder length="0.2" radius="0.25"/>
      </geometry>
    </collision>
  </link>
  <joint name="car_to_front_wheelbase" type="revolute">
    <parent link="car"/>
    <child link="front_wheelbase"/>
    <origin rpy="0 0 0" xyz="0.5 0 -0.375"/>
    <axis xyz="0 0 1"/>
    <dynamics damping="0" friction="0"/>
    <limit effort="7.0" lower="-3.14" upper="3.14" velocity="2.208932"/>
  </joint>
  <joint name="front_wheelbase_to_left" type="fixed">
    <parent link="front_wheelbase"/>
    <child link="front_wheel_left"/>
    <origin rpy="-1.5707963267948966 0 0" xyz="0 0.35 0"/>
  </joint>
  <joint name="front_wheelbase_to_right" type="fixed">
    <parent link="front_wheelbase"/>
    <child link="front_wheel_right"/>
    <origin rpy="1.5707963267948966 0 0" xyz="0 -0.35 0"/>
  </joint>
  <joint name="car_to_rear_wheelbase" type="fixed">
    <parent link="car"/>
    <child link="rear_wheelbase"/>
    <origin rpy="0 0 0" xyz="-0.5 0 -0.375"/>
  </joint>
  <joint name="rear_wheelbase_to_left" type="fixed">
    <parent link="rear_wheelbase"/>
    <child link="rear_wheel_left"/>
    <origin rpy="-1.5707963267948966 0 0" xyz="0 0.35 0"/>
  </joint>
  <joint name="rear_wheelbase_to_right" type="fixed">
    <parent link="rear_wheelbase"/>
    <child link="rear_wheel_right"/>
    <origin rpy="1.5707963267948966 0 0" xyz="0 -0.35 0"/>
  </joint>
</robot>
